webapp/components/product-info.js

```diff
@@ -146,9 +146,11 @@ const ProductInfo = (superClass) => class extends superClass {
     if (!name) {
       return;
     }
-    // TODO: Remove the special case for Servo when it has per-channel logos.
-    if (name !== 'servo' && labels) {
-      labels = new Set(labels);
+    labels = new Set(labels);
+    // Special case for Chrome nightly, which is in fact Chromium ToT:
+    if (name === 'chrome' && labels.has('nightly')) {
+      name = 'chromium';
+    } else if (name !== 'servo' && labels) {  // Servo does not have per-channel logos.
```

Comment:
Although it doesn't make a difference, just note that `if (new Set(undefined))` returns `true` here

Reply:
Good point! Reading the source code again, I think we can simply drop this, since the final outcome would be the same (an empty set would not contain any of the known channel labels).

```suggestion
    } else if (name !== 'servo') {  // Servo does not have per-channel logos.
```